cg/meta/workflow/mip_dna.py

```diff
@@ -73,3 +73,14 @@ def get_genome_build(self, case_id: str) -> str:
         """Return the reference genome build version of a MIP-DNA analysis."""
         analysis_metadata: MipAnalysis = self.get_latest_metadata(case_id)
         return analysis_metadata.genome_build
+
+    def get_data_analysis_type(self, case_id: str) -> str:
+        """Return the data analysis type of a MIP-DNA analysis
+        Patch for the typical behaviour of the AnalysisAPI function.
+        It does not raise an error with multiple analysis types.
+        """
+        case: Case = self.get_validated_case(case_id)
+        analysis_types: set[str] = {
+            link.sample.application_version.application.analysis_type for link in case.links
+        }
+        return analysis_types.pop() if analysis_types else None
```

Comment:
This function is used in mutliple places. It will now just pop the first analysis type that it gets in a "mixed" case. Is that what we want? I don't really know the flow enough to determine this.